src/tests/stub_ipc_Forwarder.cc

```diff
@@ -7,12 +7,18 @@
  */
 
 #include "squid.h"
-#include "ipc/Forwarder.h"
 
-//Avoid linker errors about Ipc::Forwarder
-void foo_stub_ipc_forwarder();
-void foo_stub_ipc_forwarder()
-{
-    Ipc::Forwarder foo(nullptr,1.0);
-}
+#define STUB_API "ipc/Forwarder.cc"
+#include "tests/STUB.h"
+
+#include "ipc/Forwarder.h"
+Ipc::Forwarder::Forwarder(Request::Pointer, double): AsyncJob("Ipc::Forwarder"), timeout(0) {STUB}
```

Comment:
We have to initialize the base class in this constructor to avoid the following compiler error. That initialization, if followed by a simple `STUB` expression, confuses Astyle -- it starts needlessly indenting the lines below it, so we have to follow with a less common (but still used) `{STUB}` expression. More details at ebea4b3.

```
tests/stub_ipc_Forwarder.cc:15:17: error: constructor for 'Ipc::Forwarder' must explicitly initialize the base class 'AsyncJob' which does not have a default constructor
```